pwnlib/rop/call.py

```diff
@@ -112,6 +112,8 @@ def __init__(self, value, address = 0):
         self.values = []
         self.address = address
         for v in value:
+            if isinstance(v, six.text_type):
+                v = context._need_bytes(v)
             if isinstance(v, (list, tuple)):
```

Comment:
Shouldn't this now be an `elif`?

Reply:
No, but the added if can be moved inside the else branch, thanks